lib/internal/util.js

```diff
@@ -10,7 +18,7 @@ exports.printDeprecationMessage = function(msg, warned) {
   if (process.throwDeprecation)
     throw new Error(msg);
   else if (process.traceDeprecation)
-    console.trace(msg);
+    console.trace(msg.replace(new RegExp(`^\\(${prefix}\\) `), ''));
```

Comment:
This is not substitution right? I stored `node` in a variable. How can that be substituted in the RegEx literal?